diff --git i/src/Makefile w/src/Makefile
index 79860b5..a695b18 100644
--- i/src/Makefile
+++ w/src/Makefile
@@ -14,20 +14,21 @@ COMPILE_FLAGS := \
 	-Wno-unused-function \
 	-I/opt/include \
 	-I$(KALDI_ROOT)/src \
 	-I$(KALDI_ROOT)/tools/openfst/include \
 	-I$(SRC_ROOT)

 LINK_FLAGS := \
 	--bind \
 	-std=c++17 \
 	-s ALLOW_MEMORY_GROWTH=1 \
+	-s DYNAMIC_EXECUTION=0 \
 	-s ERROR_ON_UNDEFINED_SYMBOLS=0 \
 	-s MODULARIZE=1 \
 	-s EXPORT_NAME=LoadVosk \
 	-s ENVIRONMENT=worker \
 	-s EXPORTED_RUNTIME_METHODS=['FS','IDBFS'] \
 	-s EXPORT_ES6=1 \
 	-s USE_ES6_IMPORT_META=0 \
 	-s EXPORTED_FUNCTIONS=['_f_init','_malloc','_free'] \
 	--post-js $(SRC_ROOT)/vosk.post.js \
 	-lidbfs.js
